### DISCUSSION TITLE
Width of text

### DISCUSSION THREAD

**Discussion by bluefander** [kdevo/osprey-delight#15] [Q&A] [answered]
Hi

Thanks for your Hugo theme!
Where can I change the width of the text of for example the about page?
Sorry, couldn't find it :/

Thanks!

**Comment by bluefander** [comment]
Found it, I guess ;)
.container {
  max-width: 43.75rem; // 700px
  margin: auto;
  padding: $vertical-rhythm * 4 0 $vertical-rhythm * 4;
}
in _layout.css

**Comment by kdevo** [comment, marked as answer]
Hello @bluefander,

Yes, that's the correct place.

However, my suggestion would be that you overload this [in a separate `_custom.scss` file within your project's root](https://github.com/kdevo/osprey-delight/blob/master/CUSTOMIZING.md#custom-scss) to avoid manually editing the theme which would make updates extra hard.

The content would then be (untested):

```css
.container {
  max-width: <YOUR_WIDTH>px
}
```

Hope this helps with the customization! I've never tested any other width personally.

---

Side note: I've converted this issue to a discussion, since issues are meant for bug- and feature-tracking only and not for any kind of support. 
Please add further questions to the [Q&A topic](https://github.com/kdevo/osprey-delight/discussions/categories/q-a). This helps me keeping everything organized. :smile:

**Reply by bluefander** [reply]
Cool, thanks!
